### ISSUE TITLE
error while loading shared libraries: libcurl-gnutls.so.4 on Centos 7

### ISSUE THREAD

**Issue by popey** [ubuntu/microk8s#630]
Trying out `microk8s` on CentOS and I can't get past the first line - `microk8s.enable istio`.

```
[snapcraft@centos7-snapcraft ~]$ microk8s.enable istio
Enabling Istio
Fetching istioctl version v1.2.2.
/snap/microk8s/743/usr/bin/curl: error while loading shared libraries: libcurl-gnutls.so.4: cannot open shared object file: No such file or directory
Failed to enable istio
```

[inspection-report-20190905_092756.tar.gz](https://github.com/ubuntu/microk8s/files/3578232/inspection-report-20190905_092756.tar.gz)

**Comment by ktsakalozos** [comment]
Hi @popey 

This looks similar to https://github.com/ubuntu/microk8s/issues/612

Could you try out the snap from `--edge`?
```
sudo snap install microk8s --classic --edge
```

**Comment by popey** [comment]
Kinda. New error.

```
curl: (77) Problem with the SSL CA cert (path? access rights?)
Failed to enable istio
```

**Comment by bsyd** [comment]
> Kinda. New error.
> 
> ```
> curl: (77) Problem with the SSL CA cert (path? access rights?)
> Failed to enable istio
> ```
Doing this on my CentOS7 box seems to resolve the same issue : 

`cp /var/lib/snapd/snap/core/current/etc/ssl/certs/ca-certificates.crt /etc/ssl/certs`

looks like the embedded curl binary has the wrong path for ca-certificates hardcoded.

**Comment by ktsakalozos** [comment]
@popey @bsyd, there should be a fix on the edge channel within the day.
`sudo snap install microk8s --classic --edge`.

Thank you @DomFleischmann for the work on this fix!

**Comment by ktsakalozos** [comment]
My tests show this fixed. The snap will be available on stable along with v1.16.1. Feel free to reopen the issue if the fix does not work for you.

Thank you for reporting this.
